Remotion/ObjectBinding/Web.UnitTests/UI/Controls/BocCustomColumnDefinitionTest.cs

```diff
@@ -52,7 +54,23 @@ public void GetComparer_WithPropertyPathNull ()
 
       var comparer = ((IBocSortableColumnDefinition)column).CreateCellValueComparer();
       Assert.That(comparer, Is.InstanceOf<BusinessObjectPropertyPathBasedComparer>());
-      Assert.That(((BusinessObjectPropertyPathBasedComparer)comparer).PropertyPath, Is.InstanceOf<NullBusinessObjectPropertyPath>());
+      Assert.That(((BusinessObjectPropertyPathBasedComparer)comparer).PropertyPath,
+          Is.InstanceOf<NullBusinessObjectPropertyPath>());
+    }
+
+    [Test]
+    public void GetValidationFailureMatcher_ReturnsInstanceOfPropertyPathValidationFailureMatcher ()
+    {
+      var column = new BocCustomColumnDefinition
+                   {
+                     CustomCell = new Mock<BocCustomColumnDefinitionCell>() { CallBase = true }.Object,
+                     OwnerControl = Mock.Of<IBocList>()
+                   };
+
+      var validationFailureMatcher = column.GetValidationFailureMatcher();
+
+      Assert.That(validationFailureMatcher, Is.InstanceOf<BusinessObjectPropertyPathValidationFailureMatcher>());
+      Assert.That(((BusinessObjectPropertyPathValidationFailureMatcher)validationFailureMatcher).PropertyPath, Is.EqualTo(column.GetPropertyPath())) ;
```

Comment:
```suggestion
      Assert.That(
          ((BusinessObjectPropertyPathValidationFailureMatcher)validationFailureMatcher).PropertyPath,
          Is.EqualTo(column.GetPropertyPath()));
```
The newline wasn't needed (I guess), the whitespace cleanup was